wgtwo/events/v0/pom.xml

```diff
@@ -50,6 +50,23 @@
                     </includes>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
this will generate an empty javadoc.jar for this module, because javadoc crashes while trying to process it.